config-linux.md

```diff
@@ -171,6 +171,14 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls (through controllers) to restrict cpu, memory, IO, pids, network and RDMA resources for the container.
 For more information, see the [kernel cgroups documentation][cgroup-v1].
 
+A runtime MAY refuse to create or start a new container, or a process inside an
```

Comment:
Do we need to be explicit that "refuse" here means "MUST return an error" and that the runtime can't choose to just silently not do anything (and still be compliant)?

(I'm not sure whether we already have similar language elsewhere.)

Reply:
Good point. I could've sworn I used the phrase "return an error" in my suggestion but I guess not. :sweat_smile: 

Yeah we usually use "return an error" explicitly.

Reply:
Yep, the phrase used elsewhere is "MUST generate an error".

Changed accordingly, PTAL @tianon @cyphar @AkihiroSuda